src/Mod/PartDesign/Scripts/Spring.py

```diff
@@ -9,10 +9,10 @@
 class MySpring:
    def __init__(self, obj):
       ''' Add the properties: Pitch, Diameter, Height, BarDiameter '''
-      obj.addProperty("App::PropertyLength","Pitch","MySpring","Pitch of the helix").Pitch=5.0
-      obj.addProperty("App::PropertyLength","Diameter","MySpring","Diameter of the helix").Diameter=6.0
-      obj.addProperty("App::PropertyLength","Height","MySpring","Height of the helix").Height=30.0
-      obj.addProperty("App::PropertyLength","BarDiameter","MySpring","Diameter of the bar").BarDiameter=3.0
+      obj.addProperty("App::PropertyLength", "Pitch", "MySpring", "Pitch of the helix").Pitch=5.0

```

Comment:
I'm surprised the linter didn't want spaces around that final "=" at the end of this line and the next few.

Reply:
I'm guessing here, but maybe the linter just saw the parentheses opening for a method and didn't expect them to close before the line's end.

Reply:
Sorry, missed that feedback chennes. I've added it and rebased in 1b488ce